<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialogClass</class>
 <widget class="QDialog" name="FindDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load route from database</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/32x32_0320/cup_add.png</normaloff>icons/32x32_0320/cup_add.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="boxQuery">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Query</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblType">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="edtValue"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblValue">
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cboxType"/>
        </item>
        <item>
         <widget class="QComboBox" name="cboxOperator">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QDialogButtonBox" name="bboxQuery">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lwResult">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bboxChoice">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionDelete_Route">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/32x32_0320/cup_delete.png</normaloff>:/icons/icons/32x32_0320/cup_delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Route</string>
   </property>
   <property name="toolTip">
    <string>Delete Route</string>
   </property>
  </action>
  <action name="actionEdit_Metadata">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/32x32_0320/cup_edit.png</normaloff>:/icons/icons/32x32_0320/cup_edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit Metadata</string>
   </property>
  </action>
  <action name="actionLoad_Route">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/icons/32x32_0320/cup_add.png</normaloff>:/icons/icons/32x32_0320/cup_add.png</iconset>
   </property>
   <property name="text">
    <string>Load Route</string>
   </property>
   <property name="toolTip">
    <string>Load Route</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
